### PR TITLE
New pages for HTML*Element.form

### PR DESCRIPTION
**Pull request by Josh-Cena**
Part of https://github.com/mdn/content/issues/35544

### CODE REVIEW

File: files/en-us/web/api/htmlbuttonelement/form/index.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## Browser compatibility
+
+{{Compat}}
+
+## See also
+
+- {{domxref("HTMLButtonElement")}}
+- {{domxref("HTMLFormElement")}}
+- {{HTMLElement("button")}}
```
Comment (estelle) [inline files/en-us/web/api/htmlbuttonelement/form/index.md:29]:
```suggestion
- {{HTMLElement("button")}}
- HTML [`form`](/en-US/docs/Web/HTML/Element/button#form) attribute
```

File: files/en-us/web/api/htmlbuttonelement/form/index.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+title: "HTMLButtonElement: form property"
+short-title: form
+slug: Web/API/HTMLButtonElement/form
+page-type: web-api-instance-property
+browser-compat: api.HTMLButtonElement.form
+---
+
+{{APIRef("HTML DOM")}}
+
+The **`form`** read-only property of the {{domxref("HTMLButtonElement")}} interface returns an {{domxref("HTMLFormElement")}} object that owns this button, or `null` if this button is not owned by any form.
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/api/htmlbuttonelement/form/index.md:11]: Not sure if we want to define that "owner" is either the form defined by the button's id attribute or the ancestor form if the attribute is not set, or that a button can only be associated with one form.

File: files/en-us/web/api/htmlfieldsetelement/form/index.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+title: "HTMLFieldSetElement: form property"
+short-title: form
+slug: Web/API/HTMLFieldSetElement/form
+page-type: web-api-instance-property
+browser-compat: api.HTMLFieldSetElement.form
+---
+
+{{APIRef("HTML DOM")}}
+
+The **`form`** read-only property of the {{domxref("HTMLFieldSetElement")}} interface returns an {{domxref("HTMLFormElement")}} object that owns this fieldset, or `null` if this fieldset is not owned by any form.
```
Comment (estelle) [inline files/en-us/web/api/htmlfieldsetelement/form/index.md:11]:
```suggestion
The **`form`** read-only property of the {{domxref("HTMLFieldSetElement")}} interface returns an {{domxref("HTMLFormElement")}} object that owns the {{HTMLElement("fieldset")}}, or `null` if the fieldset is not owned by any form.
```

File: files/en-us/web/api/htmlinputelement/form/index.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+title: "HTMLInputElement: form property"
+short-title: form
+slug: Web/API/HTMLInputElement/form
+page-type: web-api-instance-property
+browser-compat: api.HTMLInputElement.form
+---
+
+{{APIRef("HTML DOM")}}
+
+The **`form`** read-only property of the {{domxref("HTMLInputElement")}} interface returns an {{domxref("HTMLFormElement")}} object that owns this input, or `null` if this input is not owned by any form.
```
Comment (estelle) [inline files/en-us/web/api/htmlinputelement/form/index.md:11]:
```suggestion
The **`form`** read-only property of the {{domxref("HTMLInputElement")}} interface returns an {{domxref("HTMLFormElement")}} object that owns this {{htmlelement("input")}}, or `null` if this input is not owned by any form.
```

File: files/en-us/web/api/htmllabelelement/form/index.md [hunk 5]
```diff
@@ -8,20 +8,13 @@ browser-compat: api.HTMLLabelElement.form
 
 {{APIRef("HTML DOM")}}
 
-The read-only **`HTMLLabelElement.form`** property returns an
-{{domxref("HTMLFormElement")}} object which represents the form of which the label's
-associated control is a part, or null if there is either no associated control, or if
-that control isn't in a form.
+The **`form`** read-only property of the {{domxref("HTMLLabelElement")}} interface returns an {{domxref("HTMLFormElement")}} object that owns the {{domxref("HTMLLabelElement.control", "control")}} associated with this label, or `null` if this label is not associated with a control owned by a form.
```
Comment (estelle) [inline files/en-us/web/api/htmllabelelement/form/index.md:11]:
```suggestion
The **`form`** read-only property of the {{domxref("HTMLLabelElement")}} interface returns the {{domxref("HTMLFormElement")}} object that owns the form {{domxref("HTMLLabelElement.control", "control")}} the {{HTMLElement("label")}} is associated with, or `null` if the label is not associated with a control owned by a form.
```

Comment (estelle) [reply files/en-us/web/api/htmllabelelement/form/index.md:11]: do we want to mention the `form` attribute may be set on the associated form control, but not the label?

File: files/en-us/web/api/htmllegendelement/form/index.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+title: "HTMLLegendElement: form property"
+short-title: form
+slug: Web/API/HTMLLegendElement/form
+page-type: web-api-instance-property
+browser-compat: api.HTMLLegendElement.form
+---
+
+{{APIRef("HTML DOM")}}
+
+The **`form`** read-only property of the {{domxref("HTMLLegendElement")}} interface returns an {{domxref("HTMLFormElement")}} object that owns the {{domxref("HTMLFieldSetElement")}} associated with this legend, or `null` if this legend is not associated with a fieldset owned by a form.
```
Comment (estelle) [inline files/en-us/web/api/htmllegendelement/form/index.md:11]:
```suggestion
The **`form`** read-only property of the {{domxref("HTMLLegendElement")}} interface returns an {{domxref("HTMLFormElement")}} object that owns the {{domxref("HTMLFieldSetElement")}} associated with this {{htmlelement("legend")}}, or `null` if this legend is not associated with a {{htmlelement("fieldset")}} owned by a form.
```

File: files/en-us/web/api/htmlobjectelement/form/index.md [hunk 7]
```diff
@@ -8,13 +8,11 @@ browser-compat: api.HTMLObjectElement.form
 
 {{APIRef("HTML DOM")}}
 
-The **`form`** read-only property of the
-{{domxref("HTMLObjectElement")}} interface returns a {{domxref("HTMLFormElement")}}
-representing the object element's form owner, or null if there isn't one.
+The **`form`** read-only property of the {{domxref("HTMLObjectElement")}} interface returns an {{domxref("HTMLFormElement")}} object that owns this object, or `null` if this object is not owned by any form.
```
Comment (estelle) [inline files/en-us/web/api/htmlobjectelement/form/index.md:11]:
```suggestion
The **`form`** read-only property of the {{domxref("HTMLObjectElement")}} interface returns an {{domxref("HTMLFormElement")}} object that owns this {{htmlelement("object")}}, or `null` if this object element is not owned by any form.
```
"element" is needed to differentiate `<object>` from object.

File: files/en-us/web/api/htmloptionelement/form/index.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+title: "HTMLOptionElement: form property"
+short-title: form
+slug: Web/API/HTMLOptionElement/form
+page-type: web-api-instance-property
+browser-compat: api.HTMLOptionElement.form
+---
+
+{{APIRef("HTML DOM")}}
+
+The **`form`** read-only property of the {{domxref("HTMLOptionElement")}} interface returns an {{domxref("HTMLFormElement")}} object that owns the {{domxref("HTMLSelectElement")}} associated with this option, or `null` if this option is not associated with a select owned by a form.
```
Comment (estelle) [inline files/en-us/web/api/htmloptionelement/form/index.md:11]:
```suggestion
The **`form`** read-only property of the {{domxref("HTMLOptionElement")}} interface returns an {{domxref("HTMLFormElement")}} object that owns the {{domxref("HTMLSelectElement")}} associated with this option, or `null` if this {{htmlelement("option")}} is not associated with a {{htmlelement("select")}} owned by a form.
```

Comment (estelle) [reply files/en-us/web/api/htmloptionelement/form/index.md:11]: do options within a `<datalist>` have a `form` property?

File: files/en-us/web/api/htmloutputelement/form/index.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+title: "HTMLOutputElement: form property"
+short-title: form
+slug: Web/API/HTMLOutputElement/form
+page-type: web-api-instance-property
+browser-compat: api.HTMLOutputElement.form
+---
+
+{{APIRef("HTML DOM")}}
+
+The **`form`** read-only property of the {{domxref("HTMLOutputElement")}} interface returns an {{domxref("HTMLFormElement")}} object that owns this output, or `null` if this output is not owned by any form.
```
Comment (estelle) [inline files/en-us/web/api/htmloutputelement/form/index.md:11]:
```suggestion
The **`form`** read-only property of the {{domxref("HTMLOutputElement")}} interface returns an {{domxref("HTMLFormElement")}} object that owns this {{htmlelement("output")}}, or `null` if this output is not owned by any form.
```

File: files/en-us/web/api/htmlselectelement/form/index.md [hunk 10]
```diff
@@ -6,45 +6,13 @@ page-type: web-api-instance-property
...
 ---
 
-{{ APIRef("HTML DOM") }}
+{{APIRef("HTML DOM")}}
 
-The **`HTMLSelectElement.form`** read-only property returns a
-{{domxref("HTMLFormElement")}} representing the form that this element is associated
-with. If the element is not associated with a {{HTMLElement("form")}} element, then
-it returns `null`.
+The **`form`** read-only property of the {{domxref("HTMLSelectElement")}} interface returns an {{domxref("HTMLFormElement")}} object that owns this select, or `null` if this select is not owned by any form.
```
Comment (estelle) [inline files/en-us/web/api/htmlselectelement/form/index.md:11]:
```suggestion
The **`form`** read-only property of the {{domxref("HTMLSelectElement")}} interface returns an {{domxref("HTMLFormElement")}} object that owns this {{htmlelement("select")}}, or `null` if this select is not owned by any form.
```

File: files/en-us/web/api/htmltextareaelement/form/index.md [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+title: "HTMLTextAreaElement: form property"
+short-title: form
+slug: Web/API/HTMLTextAreaElement/form
+page-type: web-api-instance-property
+browser-compat: api.HTMLTextAreaElement.form
+---
+
+{{APIRef("HTML DOM")}}
+
+The **`form`** read-only property of the {{domxref("HTMLTextAreaElement")}} interface returns an {{domxref("HTMLFormElement")}} object that owns this text area, or `null` if this text area is not owned by any form.
```
Comment (estelle) [inline files/en-us/web/api/htmltextareaelement/form/index.md:11]:
```suggestion
The **`form`** read-only property of the {{domxref("HTMLTextAreaElement")}} interface returns an {{domxref("HTMLFormElement")}} object that owns the {{htmlelement("textarea")}}, or `null` if this textarea is not owned by any form.
```

**Review by estelle — COMMENTED** [review]
minor edits and a question

File: files/en-us/web/api/htmloptionelement/form/index.md (hunk 8 above)
Comment (Josh-Cena) [reply files/en-us/web/api/htmloptionelement/form/index.md:11]: No I don't think so. They can only be associated via the parent `<select>`.

File: files/en-us/web/api/htmlbuttonelement/form/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Josh-Cena) [reply files/en-us/web/api/htmlbuttonelement/form/index.md:11]: I think the precise definition of "owner", if we are to have one, should be in a guide article.

**Review by estelle — APPROVED** [review]
Thanks! 🎉

**Pull request merged**